testutils/block.go

```diff
@@ -0,0 +1,46 @@
+package testutils
+
+import (
+	"github.com/renproject/hyperdrive/block"
+	"github.com/renproject/hyperdrive/sig"
+)
+
+func SignBlock(blk block.Block, signer sig.SignerVerifier) *block.SignedBlock {
+	signedBlock, _ := blk.Sign(signer)
+	return &signedBlock
+}
+
+func GenerateSignedPreVote(signedBlock block.SignedBlock, signer sig.SignerVerifier) block.SignedPreVote {
+	preVote := block.PreVote{
+		Block:  &signedBlock,
+		Height: signedBlock.Height,
+	}
+	signedPreVote, _ := preVote.Sign(signer)
```

Comment:
I really cannot express this enough, and it's something we should all already be very diligent about. Never ignore the error.